ui/components/layouts/dashboard.js

```diff
@@ -106,3 +107,13 @@ export default function ({ children }) {
     </div>
   )
 }
+
+export default function ({ children }) {
+  return (
+    <AuthRequired>
```

Comment:
should we move the `AuthRequired` to `_app.js` so we don't need to wrap it in inside the dashboard and fullscreen components?

Reply:
Great question – I thought about this, but `_app.js` is a component that's rendered for every page, whereas `AuthRequired` should only be wrapping a subset of pages – the idea here that all `Dashboard` and `Fullscreen` layouts need auth – but you could do it on an individual page basis too